docs/getting-started/cases-req.asciidoc

```diff
@@ -34,7 +39,12 @@ a|
 * **All** for the *Cases* feature under *Security*
 * **All** for the *{connectors-feature}* feature under *Management*
 
-NOTE: Roles without **All** *{connectors-feature}* feature privileges cannot create, add, delete, or modify case connectors.
+[NOTE]
+====
+Roles without **All** *{connectors-feature}* feature privileges cannot create, add, delete, or modify case connectors.
```

Comment:
No issue with the sentence structure. This suggestion is mainly to improve clarity. The way the text is styled, it sort of looks like we're telling users that they need feature privs for the All Actions and Connectors feature (see example below). 

<img width="400" alt="Screenshot 2024-01-08 at 3 29 02 PM" src="https://github.com/elastic/security-docs/assets/79124755/78fa419d-594e-4bc5-a65e-e9fd2a6a90e1">

```suggestion
Roles without **All** privileges for the *{connectors-feature}* feature cannot create, add, delete, or modify case connectors.
```